### PR TITLE
Fix `ApprovalHash` Incorrect Storage of task `020-holocene-system-config-upgrade-multi-chain`

### PR DESCRIPTION
**Pull request by Ethnical**
**Description**

 @geoknee Discover and Inform me of an inconsistency between the `approveHash` storage calculation from the task `020-holocene-system-config-upgrade-multi-chain`. 

After investigation, I missed the change of `nestedHash` value (so the value were calculated on the incorrect value but were matching the `VALIDATION.md` file that has incorrect values...).

Now, the value are matching the simulation correctly for example for security council below:

**State Precomputed**
ddda62b07befa8a55ccccfc521a57485360a393dbf0a4e2d93d1c3d4870d41f2

**Tenderly Simulation**
d607c91ff674481c6e8707e9f48b144be8fa47e07c2eee69ad675726e5d484ec

### CODE REVIEW

**Review by geoknee — APPROVED** [review]

**Review by raffaele-oplabs — APPROVED** [review]
Approved as a codeowner to unblock. I trust @geoknee review who have already approved this PR.

**Pull request merged**
